www/source/partials/docs/_reference-plan-settings.html.md.erb

```diff
@@ -204,6 +206,45 @@ In this example, the corresponding default.toml file would have the following ke
 ### pkg\_upstream\_url
 **Optional**. An upstream project homepage or website URL.
 
-  ~~~
-  pkg_upstream_url=https://github.com/myrepo
-  ~~~
+~~~
+pkg_upstream_url=https://github.com/myrepo
+~~~
+
+## Composite-specific Settings
+### pkg\_type
+**Required**. String that denotes a composite package. Must be set to `"composite"` otherwise Habitat will treat the composite package as a normal package. 
+
+~~~
+pkg_type="composite"
+~~~
+
+### pkg\_services
+**Required**. Array that contains all of the services in your composite application. 
+
+> **Note** You must list at least two services and if you list a package that is not a service (for example, it does not have a run hook), it will trigger a build error for the composite package.
+
+~~~
+pkg_services=(
+    cm/sample-node-app
+    cm/composite-example-api-proxy
+)
+~~~
+
+### pkg\_bind\_map
+**Required**. Array that contains any service bindings between composite services. Additional bindings can be added separated by spaces. The format is `[consumer_service]:"bind_key:origin/producerservice"`. 
```

Comment:
Strictly speaking, this is not currently required; I think it would be very rare to _not_ have a `pkg_bind_map` set, though.